tesla_ble_mqtt/Dockerfile

```diff
@@ -19,10 +19,13 @@ RUN apk add --no-cache \
   bluez \
   bluez-deprecated \
   mosquitto-clients \
-  openssl
-  
-# Copy data for add-on
+  openssl \
+  sed \
+  vim
```

Comment:
do we need to keep vim there?

Reply:
Now that I can easily deploy a local add-no, I'll remove it.

The history; I use it to edit files directly in the container. The included vi has very limited in terms of search & replace w/ regex, no color, revert changes yes but no forward revert. You can't imagine how often I reverted changes and found out I couldn't go back to what I was doing (un-revert).